### ISSUE TITLE
ytdl: Start avc1 faster?

### ISSUE THREAD

**Issue by kevinlekiller** [mpv-player/mpv#2395]
Low priority issue.

When requesting avc1 with `--ytdl-format=137+171` for example, the whole video is downloaded before starting, on a 2 hour video this can take a long time.

Decoding avc1 is much easier than vp9, takes about 2x-3x less power according to `turbostat --debug` (possibly because my system doesn't have hardware decode for vp9 or something).

My thinking was, because it's mp4 it needs to be fully downloaded? If this is the case, you can close the issue.

The same issue happens for aac audio also, `--ytdl-format=137+140`, now the whole audio is also downloaded, which extends the load time further, but this is a not an issue for me since the power it takes to decode audio is insignificant compared to the video.

Thanks.

**Comment by wm4** [comment]
Use ffmpeg git master, where this is fixed.

**Comment by kevinlekiller** [comment]
Didn't change anything, but not a big deal, I'll stick with vp9

**Comment by wm4** [comment]
It should have. Are you sure you're using recent ffmpeg git master?

**Comment by kevinlekiller** [comment]
Yeah, I built with mpv-build  ./rebuild -j5

**Comment by kevinlekiller** [comment]
Actually, my bad, mpv_build is on 2.8 by default

**Comment by kevinlekiller** [comment]
Yep that did it, thanks for the support!
